test/python/circuit/test_extensions_standard.py

```diff
@@ -1371,105 +1370,5 @@ def test_cswap_reg_reg_inv(self):
         self.assertEqual(instruction_set[1].qubits, (self.qr[1], self.qr2[1], self.qr3[1]))
         self.assertEqual(instruction_set[2].operation.params, [])
 
-
-class TestStandardMethods(QiskitTestCase):
```

Comment:
can we do these tests with PhaseOracle w/o tweedledum ?

Reply:
Yes, actually tweedledum was not really needed there. I returned the test.